CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # SAS Viya Monitoring for Kubernetes
 
-## Unreleased
+## Version 1.2.23 (19MAR2024)
```

Comment:
Please copy the [ANNOUNCEMENT] entry from last month's release (announcing the breaking change) and include in the **metrics** section of this month's CHANGELOG as well.  I think the announcement says the change is scheduled for this release (1.2.23) so please update that to say we are tentatively scheduling it for the next release (1.2.24).